indico/modules/events/registration/notifications.py

```diff
@@ -33,11 +36,21 @@ def notify_invitation(invitation, email_subject, email_body, from_address):
 
 
 def _notify_registration(registration, template, to_managers=False):
+    attachments = None
+    regform = registration.registration_form
+    tickets_handled = values_from_signal(signals.event.is_ticketing_handled.send(regform), single_value=True)
+    if (not to_managers and
+            regform.tickets_enabled and
+            regform.ticket_on_email and
+            not (tickets_handled and all(tickets_handled)) and
```

Comment:
@meluru I think `any(tickets_handled)`  would be more appropriate here. In the rare case that multiple plugins handle tickets (do we even support this) it should be enough if one of these plugins intercept ticketing..